src/utils/colors.ts

```diff
@@ -0,0 +1,6 @@
+export function getColorFromId(id: string, luminance: number): string {
```

Comment:
Should we add a comment about which color mood this is supposed to produce ? like pastel ?